package.json

```diff
@@ -50,6 +50,7 @@
     "start:dev_api": "concurrently --raw \"cd packages/app && yarn start:dev_api\" \"cd packages/common && yarn start\" \"cd packages/components && yarn start\"",
     "start:dynamic": "lerna run dev --scope dynamic-pages --stream",
     "start:fast": "concurrently --raw \"cd packages/app && yarn start\" \"cd packages/common && yarn start\" \"cd packages/components && yarn start\"",
+    "start:fast:stream": "cross-env STAGING_STREAM=1 concurrently --raw \"cd packages/app && yarn start\" \"cd packages/common && yarn start\" \"cd packages/components && yarn start\"",
```

Comment:
I think `STAGING_STREAM` is kind of confusing, because `STAGING` and `STREAM` mean pretty much the same thing. 🙂 Maybe `DEV_STREAM` is better, i.e. "develop on stream"?

Reply:
Hmmmmm, that's right. But I'd interpret `DEV_STREAM` as the development server on stream. Pretty hard to get a good name here. I think we should still include `STAGING` in the name, since quite some people don't know that `STREAM` is actually staging.

Reply:
`DEV_STAGING`, maybe? Or turn it around and say `DEV_PROD=1` and make stream default?

Reply:
I think we should stick with one term... if `STREAM` is our staging environment, let us just stick with that all over. Does not really matter what we call it, as long as it is consistent. Or are we referencing `STAGING` somewhere else?

Reply:
Yeah we have another env var called `STAGING`, otherwise I would've chosen that env var.

Reply:
What if we would change the var name to `STAGING_API`?

Reply:
`USE_STAGING_API`